src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookConnector.java

```diff
@@ -49,13 +53,30 @@ public WebhookConnector(String projectKey) {
 				ConfigPersistenceManager.getEnabledWebhookTypes(projectKey));
 	}
 
-	public boolean sendElementChanges(KnowledgeElement changedElement) {
+	public boolean sendElement(KnowledgeElement sendElement) {
+		return sendElement(sendElement, "new");
+	}
+
+	public boolean sendElement(KnowledgeElement sendElement, String event) {
+		if (sendElement == null || event == null) {
+			return false;
+		}
 		boolean isSubmitted = false;
-		if (!checkIfDataIsValid(changedElement)) {
+
+		if (!checkIfDataIsValid(sendElement)) {
 			return isSubmitted;
 		}
-		List<KnowledgeElement> rootElements = getWebhookRootElements(changedElement);
-		isSubmitted = postKnowledgeTrees(rootElements);
+
+		switch (type) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)